Release/src/http/client/http_client_winhttp.cpp

```diff
@@ -15,14 +15,55 @@
 #include "stdafx.h"
 
 #include <atomic>
+#include <Wincrypt.h>
 
 #include "cpprest/http_headers.h"
+#include "cpprest/details/x509_cert_utilities.h"
```

Comment:
This will cause a problem when building with CPPREST_EXCLUDE_WEBSOCKETS.

Use `cmake -DCPPREST_EXCLUDE_WEBSOCKETS=ON` to test.

Reply:
Specifically, the file is guarded to not pull in anything if that macro is set on windows; however simply removing the guard doesn't work because we need to also guard against pulling in the boost asio ssl headers.